app/src/main/java/ch/epfl/qedit/backend/auth/MockAuthService.java

```diff
@@ -34,10 +34,10 @@ public IdlingResource getIdlingResource() {
         return idlingResource;
     }
 
-    /** Simply make the current thread wait 2 second */
-    private static void wait2second() {
+    /** Simply make the current thread wait 0.5 seconds, to do as if the request takes time */
+    private static void fakeWait() {
         try {
-            Thread.sleep(2000);
+            Thread.sleep(500);
```

Comment:
You do this to speed up our tests, am I right?

Reply:
Yes, exactly. And also to speed up the demonstration that will be using the mocks.